exporter/awscloudwatchlogsexporter/go.mod

```diff
@@ -9,3 +9,7 @@ require (
 	go.opentelemetry.io/collector/model v0.30.0
 	go.uber.org/zap v1.18.1
 )
+
+replace go.opentelemetry.io/collector => go.opentelemetry.io/collector v0.29.1-0.20210716020257-4d8e3082465d
+
+replace go.opentelemetry.io/collector/model => go.opentelemetry.io/collector/model v0.0.0-20210716020257-4d8e3082465d
```

Comment:
This is confusing, why these changes?

Reply:
Why not 0.30.0 something?

Reply:
this is a good question. `go get` returned an errors about the versions not existing when I tried w/ 0.30.1

Reply:
This is the versions I ended up with from using main

```
go get go.opentelemetry.io/collector@main
go get: downgraded go.opentelemetry.io/collector v0.30.0 => v0.29.1-0.20210716213518-a5a838ed3569

go get go.opentelemetry.io/collector/model@main
go get: downgraded go.opentelemetry.io/collector/model v0.30.0 => v0.0.0-20210716213518-a5a838ed3569
```